velox/serializers/PrestoSerializer.cpp

```diff
@@ -2934,7 +2964,7 @@ void estimateWrapperSerializedSize(
 }
 
 template <TypeKind Kind>
-void estimateConstantSerializedSize(
+void estimateFlattenedConstantSerializedSize(
```

Comment:
Since you are already deep into looking at these helper functions, it would be great if you can add short comments to some of them (maybe as a follow up). For eg, some of their names can get confusing like "estimateWrapperSerializedSize" due the overuse of words like wrapper, wrapped, flattened etc. in different contexts. so would be great to get a confirmation of what it actually does without reading through the implementation.